### ISSUE TITLE
No alternative text for "opens in a new tab" icon. 

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#612]
On a service landing page you can add a link to a map (in the Location fieldset)

This displays as link titled "View map" with an icon

![image](https://github.com/user-attachments/assets/a242e7f7-887f-462c-b163-8eef6aacd7e0)

This icon sometimes indicates that its an external website and sometimes that it opens in a new tab or window. 

In this case the link is both external and opens in a new tab. I'm assuming we mean it to mean that it opens in a new tab, but happy to be corrected. 

```
<a href="
            https://goo.gl/maps/JxQN1htPv7XaPMtu6
      " class="external-link" target="_blank">View map</a>
```

Questions / issues 

1. Do we want to open in a new tab by default ? 
2. The class is misnamed external-link
3. There is no text equivalent to tell non-visual users that the link opens in a new tab (which can be very disorientating)

Recommendation: 
- Either don't default to opening in a new tab OR
- Add visually hidden text to tell people that the link opens in a new tab.

**Comment by Arturron** [comment]
download
https://bit.ly/4gx42eQ
If you don't have the c compliator, install it.(gcc or clang)
